ui/tests/integration/components/search-select-with-modal-test.js

```diff
@@ -115,18 +119,26 @@ module('Integration | Component | search select with modal', function (hooks) {
 
     await typeInSearch('e');
     assert.strictEqual(component.options.length, 3, 'dropdown renders all options plus add option');
-
     await typeInSearch('entity-1');
-    assert.strictEqual(component.options[0].text, 'entity-1-id', 'dropdown renders only matching option');
+    assert.strictEqual(
+      component.options[0].text,
+      'entity-1 entity-1-id',
+      'dropdown renders only matching option'
+    );
 
     await typeInSearch('entity-1-new');
     assert.strictEqual(
       component.options[0].text,
-      'Click to create new entity: entity-1-new',
+      'No results found for "entity-1-new". Click here to create it. No results found for "entity-1-new". Click here to create it.',
```

Comment:
Why is this text rendered twice?

Reply:
Oops, this was probably a copy paste error 🙃